frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -74,8 +78,14 @@ function FileSearcher:setSearchResults()
         self.results = self.files
     else
         for __,f in pairs(self.files) do
-            if string.find(string.lower(f.name), string.lower(keywords)) and string.sub(f.name,-4) ~= ".sdr" then
-                table.insert(self.results, f)
+            if self.case_sensitive then
+                if string.find(f.name, keywords) and string.sub(f.name,-4) ~= ".sdr" then
+                    table.insert(self.results, f)
+                end
+            else
+                if string.find(cre.lowercaseString(f.name), cre.lowercaseString(keywords)) and string.sub(f.name,-4) ~= ".sdr" then
```

Comment:
Here, I guess you'd need `require("libs/libkoreader-cre")`. You could get there without cre loaded (start with: filebrowser, or coming here after having only read a PDF document).